mico-core/src/main/java/io/github/ust/mico/core/model/MicoServicePort.java

```diff
@@ -39,6 +39,7 @@
 
 /**
  * Represents a basic port with a port number and port type (protocol).
+ * Is used also as a DTO for requests and responses.
```

Comment:
There is no DTO `MicoServicePort`. Is this on purpose? It is quite confusing, because besides `MicoLabel` it is the only domain class that is also used as a request and response DTO. Therefore this class is the only one that needs the annotation `@JsonProperty(access = JsonProperty.Access.READ_ONLY)` for the ID property. This would not be required, if there would be special request DTO.

Reply:
`MicoServicePortRequestDTO` and `MicoServicePortResponseDTO` added.